src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/PersonEmail/PersonEmail.tsx

```diff
@@ -104,35 +116,43 @@ export const PersonEmail: React.FC<PersonEmailProps> = ({ formikProps }) => {
                       <Grid container spacing={3}>
                         <Grid item xs={6}>
                           <ContactInputField
+                            destroyed={emailAddress.destroy ?? false}
```

Comment:
This always has a default of false where I declare it, but the type says it could be false or undefined. So I have to do this here to satisfy the type.